src/sampling/homogeneous.jl

```diff
@@ -40,6 +40,24 @@ function sample(rng::AbstractRNG, triangle::Triangle{Dim,T}, method::Homogeneous
   (randpoint() for _ in 1:(method.size))
 end
 
+function sample(rng::AbstractRNG, ball::Ball{Dim,T}, method::HomogeneousSampling) where {Dim,T}
+  if Dim == 2
+    function randpoint2()
+      u₁, u₂ = rand(rng, T, 2)
+      ball(sqrt(u₁), u₂)
+    end
+    (randpoint2() for _ in 1:(method.size))
+  elseif Dim == 3
+    function randpoint3()
+      u₁, u₂, u₃ = rand(rng, T, 3)
+      ball(u₁^(1/3), acos(1 - 2u₂) / π, u₃)
```

Comment:
also, these expressions in terms of u1, u2, u3. We need to stretch these dimensions so that points are distributed "homogeneously"?

Reply:
The same as in 2D. For example, in the 3D case, it can be shown that the marginal distribution of the radio is $3r^2/R^3$ which has a CDF as $r^3/R^3$ so it can be simulated using $Rr^{1/3}$.